From akpm@osdl.org Tue Apr 18 21:19:10 2006
Message-Id: <200604190419.k3J4J1tF008521@shell0.pdx.osdl.net>
Subject: W1: fix dependencies of W1_SLAVE_DS2433_CRC
To: greg@kroah.com
Cc: akpm@osdl.org, jean-luc.leger@dspnet.fr.eu.org
From: akpm@osdl.org
Date: Tue, 18 Apr 2006 21:18:20 -0700


From: Jean-Luc Leger <jean-luc.leger@dspnet.fr.eu.org>

Default values for boolean and tristate options can only be 'y', 'm' or 'n'. 
This patch fixes dependencies of W1_SLAVE_DS2433_CRC.

Signed-off-by: Jean-Luc Leger <jean-luc.leger@dspnet.fr.eu.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/w1/slaves/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/w1/slaves/Kconfig
+++ gregkh-2.6/drivers/w1/slaves/Kconfig
@@ -28,7 +28,7 @@ config W1_SLAVE_DS2433
 
 config W1_SLAVE_DS2433_CRC
 	bool "Protect DS2433 data with a CRC16"
-	depends on W1_DS2433
+	depends on W1_SLAVE_DS2433
 	select CRC16
 	help
 	  Say Y here to protect DS2433 data with a CRC16.
